test-requirements.txt

```diff
@@ -0,0 +1,6 @@
+-r requirements.txt
+pyflakes==2.2.0
+flake8==2.6.0
+pylint==2.5.2
+pytype==2020.6.1
+faucet==1.9.43
```

Comment:
What is the reason for locking the FAUCET version?

If we do this, I think we still want to run CI against the latest and greatest FAUCET.

Reply:
FAUCET itself is a dependency like all the other things - it'd be advantageous to have a predictable environment (and it is consistent with the other FAUCET-dependant projects like Poseidon).

Reply:
Actually, I changed my mind - I will comply with all the requests because I need the PR in.